README.md

```diff
@@ -13,6 +13,16 @@
 
 A community repository for Ansible Playbook of OpenSearch Project.
 
+## Version and Branching
+As of now, this helm-charts repository maintains 3 branches:
```

Comment:
@peterzhuamazon must be `ansible-playbook` instead of `helm-charts`

Reply:
fixed.